### PR TITLE
Pull jobs and logs into the Build Jobs tab

### LINKED ISSUE #3084: Pull jobs and logs into the Build Jobs tab
Given then new IA, we need to relocate the components that comprise this feature:

https://chef.invisionapp.com/d/#/console/10957507/249011621/preview

### PR DESCRIPTION
**Pull request by cnunciato**
This change brings the build-related componentry (lists, logs, etc.) into the new Build Jobs tab, tweaks a few names for consistency, and adds some logic for conditionally displaying the package sidebar. 

Signed-off-by: Christian Nunciato <cnunciato@chef.io>

Fixes #3084.

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: components/builder-web/app/package/package/package.component.ts [hunk 1]
```diff
@@ -38,4 +43,19 @@ export class PackageComponent implements OnDestroy {
             name: this.name
         };
     }
+
+    onRouteActivate(routedComponent) {
```
Comment (elliott-davis) [inline components/builder-web/app/package/package/package.component.ts]: Is this checking to see if the currently routed component is in this array, and if so, enable the sidebar?

Comment (cnunciato) [reply components/builder-web/app/package/package/package.component.ts]: Yep. Specifically it's being used to apply a class that shows the sidebar (as opposed to re-rendering it), but yes, that's the gist.

**Comment by cnunciato** [comment]
@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/272601437).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/272601437).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**